packaging/pkcs11-provider.spec

```diff
@@ -54,8 +55,8 @@ make check || if [ $? -ne 0 ]; then cat tests/*.log; exit 1; fi;
 
 %files
 %license COPYING
-%{_mandir}/man7/*
-%doc README
+%{_mandir}/man7/%{name}*
```

Comment:
Why this change?
The man page will not match this
name = pkcs11-provider
manpage = provider-pkcs11.7

Reply:
It  was a suggestion on the fedora-review bug that too unspecified wildcard was used.
If we don't want to align the manpage to name, then I guess we can let it be as is.
I have reverted the change.